File: twiml/voice/sip/sip-7/sip-7.5.x.php
<?php
require_once './vendor/autoload.php';
use Twilio\Twiml;

$response = new Twiml();
$dial = $response->dial();
$dial->sip('kate@example.com', ['username' => 'admin', 'password' => '1234']);

echo $response;

File: twiml/voice/sip/sip-7/sip-7.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var dial = new Dial();
        dial.Sip(new Uri("kate@example.com", UriKind.Relative),
            username: "admin", password: "1234");
        response.Append(dial);

        Console.WriteLine(response.ToString());
    }
}
